src/main/java/io/crate/testing/Utils.java

```diff
@@ -41,7 +48,59 @@ static int randomAvailablePort() {
         }
     }
 
-    static void log(String message, Object ... params) {
+    static void log(String message, Object... params) {
         System.out.println(String.format(Locale.ENGLISH, message, params));
     }
+
+    public static void deletePath(Path path) throws Exception {
```

Comment:
we can use common io FileUtils to delete files in the path recursively, but i didn't want to add additional dependencies, just for one method call. With plain java we cannot delete non-empty dirs, thus this file visitor for deleting dir recursively was implemented.